hyperswitch/PMMViewController.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+import WebKit
+import Combine
+import SwiftUI
+
+class PaymentMethodManagementViewController: UIViewController {
+    @ObservedObject var hyperViewModel = HyperViewModel()
+    private var paymentSession: PaymentSession?
+    private var cancellables = Set<AnyCancellable>()
+    private var topElement: UIView?
+    
+    private func setupPaymentWidget(onAddPaymentMethod: @escaping () -> Void) {
+        guard hyperViewModel.paymentSession != nil else { return }
+        
+        let paymentWidget = PaymentMethodManagementWidget(
+            ephemeralKey: PaymentSession.ephemeralKey ?? "",
```

Comment:
why accessing PaymentSession.ephemeralKey from internal code, use the model to get this

Reply:
removed ephemeral key from the init call param, as it was already defined while creating `PaymentSession`.